cmd/rook/ceph/agent.go

```diff
@@ -63,7 +63,7 @@ func startAgent(cmd *cobra.Command, args []string) error {
 	agent := agent.New(context)
 	err = agent.Run()
 	if err != nil {
-		rook.TerminateFatal(fmt.Errorf("failed to run rook ceph agent. %+v\n", err))
```

Comment:
Just to clarify, the go linting that my editor ran was giving me warnings about ending error messages in newlines and/or beginning them with capitalized words.